site/src/pages/LoginPage/LoginPage.test.tsx

```diff
@@ -89,4 +90,18 @@ describe("LoginPage", () => {
     await screen.findByText(Language.passwordSignIn)
     await screen.findByText(Language.githubSignIn)
   })
+
+  it("redirects to the setup page if there is no user", async () => {
+    // Given
+    jest.spyOn(API, "hasFirstUser").mockResolvedValueOnce(true)
```

Comment:
potentially dumb q: does the handler in `handlers.ts` not take care of this? Why do we have to mock out the value here?

Reply:
Done!